app/services/jadu/claim.rb

```diff
@@ -43,8 +43,20 @@ def client
 
     def attachments
```

Comment:
This issue is gnarly as f***. Confirmed on dev and prod, i.e. file and s3 backends.